scripts/vmbuild.rb

```diff
@@ -110,6 +110,7 @@ def verify_run(output)
     $log.info "Building for #{target}:"
 
     tdl_name = target.name == "azure" ? "base_azure.tdl" : "base.tdl"
+    base_file   = CFG_DIR.join("base.json.azure") if target.name == "azure"
```

Comment:
Is the line above also temporary? Or is that something more permanent? If they are in the same kind of situation, can we merge the identical conditionals?

Reply:
That's temporary too, I'll be reverting this entire PR.

Reply:
The line above is not in this PR. We have two conditionals that say `if target.name == "azure"` can we DRY that up?

Reply:
oh, you mean tdl_name.  I didn't DRY that up because I'll be removing this PR (hopefully soon), but I can certainly do that...

Reply:
That's fine I was only suggesting it if both azure special cases were the same special case :laughing: